core/lib/object_store/src/objects.rs

```diff
@@ -63,6 +67,62 @@ macro_rules! serialize_using_bincode {
     };
 }
 
+/// Derives [`StoredObject::serialize()`] and [`StoredObject::deserialize()`] using
+/// the `json` (de)serializer. Should be used in `impl StoredObject` blocks.
```

Comment:
???

Reply:
some leftovery, which my IDE tried to parse as documentation and hid behind "Not availablee", thanks for spotting!